index.js

```diff
@@ -52,31 +52,31 @@ function buildFetch(behavior) {
   }
   const requestTimeFn = behavior.requestTimeFn || defaultRequestTimeFn;
 
-  if (behavior.hasOwnProperty("clone")) {
+  if (Object.prototype.hasOwnProperty.call(behavior, "clone")) {
```

Comment:
Not sure why this rule is necessary in a backend library? Or we could use `"clone" in behaviour`.

Reply:
In this particular case the rule isn't necessary, but is there a reason to not use Object.prototype.hasOwnProperty.call ?

Reply:
there is no reason if you are a developing a backend project. Though we could argue that this rule will nudge devs to use in operator.

Reply:
The rule applies to any project where the input may come from a client, including backend projects. However in this particular case it is not necessary but on the other-hand it doesn't hurt to use it either.

Write